group/templates/group/groupDetail.html

```diff
@@ -45,215 +43,220 @@ <h3 style="display:inline">,&nbsp</h3>
-                </tr>
-              </thead>
-              <tbody>
-                {% for running_contest in rc_list %}
-                  {% if running_contest.is_homework %}
-                    <tr style="background-color:#F1F7EA">
-                      <td>{{ running_contest.cname }}</td>
-                      <td>{{ running_contest.start_time }}</td>
-                      <td>{{ running_contest.end_time }}</td>
-                    </tr>
-                  {% else %}
-                    <tr style="background-color:#E3E8E3">
-                      <td>{{ running_contest.cname }}</td>
-                      <td>{{ running_contest.start_time }}</td>
-                      <td>{{ running_contest.end_time }}</td>
+  <div class="container-fluid">
+    <div class="Description" style="float:left">
+      <p style="font-size:16px; color:gray; word-wrap:break-word">
+        {{ group_description }}
+      </p>
+    </div>
+  </div>
+  <hr>
+  <div class="col-lg-offset-0 col-lg-12">
+    <div class="contest">
+      <div style="margin-bottom:15px">
+        <h4 style="text-decoration:underline;color:gray">Traced Contest</h4>
+      </div>
+      <div class="panel-group" id="accordion">
+        <div class="panel panel-success">
+          <div class="panel-heading">
+            <h4 class="panel-title">
+              <a data-toggle="collapse" data-parent="#accordion" 
+                 href="#runningContestPanel">Running Contest</a>
+            </h4>
+          </div>
+          <div id="runningContestPanel" class="panel-collapse collapse in">
+            <div class="panel-body">
+              <table class="table table-hover table-striped">
+                <thead>
+                  <tr>
+                    <th>ID</th>
+                    <th>Title</th>
+                    <th>Start Time</th>
+                    <th>End Time</th>
+                  </tr>
+                </thead>
+                <tbody>
+                  {% for running_contest in running_contest_list %}
+                    <tr>
+                      <td>{{ running_contest.id }}</td>
+                      <td>
+                        <a href="{% url 'contest:contest' running_contest.id %}">
+                          {{ running_contest.cname }}
+                        </a>
+                      </td>
+                      <td>{{ running_contest.start_time|date:"Y/m/d H:i:s" }}</td>
+                      <td>{{ running_contest.end_time|date:"Y/m/d H:i:s" }}</td>
                     </tr>
-                  {% endif %}
-                {% endfor %}
-              </tbody>
-            </table>
-            <a class="btn btn-sm btn-default" style="float:right"
-               onclick="window.open('{% url 'group:viewall_contest' group_id %}', '', config='height=500,width=800');">view all</a>
+                  {% endfor %}
+                </tbody>
+              </table>
+              <a class="btn btn-sm btn-default" style="float:right"
```

Comment:
There are 3 "view all" button, their only difference url. Is there any possibility to put them into css file, so that we can eliminate redundant code?(I'm not sure, just asking)